scripts/install-mender.sh

```diff
@@ -202,19 +202,29 @@ get_deps() {
 add_repo() {
     curl -fsSL $REPO_URL/gpg | apt-key add -
 
-    repo="deb [arch=$ARCH] $REPO_URL $CHANNEL main"
-
-    echo "Checking if mender sources already exist in '/etc/apt/sources.list'..."
-    if grep -F "$repo" /etc/apt/sources.list; then
-        echo "Removing the old mender debian source list from /etc/apt/sources.list..."
-        if ! sed -i.bak -e "\,$REPO_URL,d" /etc/apt/sources.list; then
-            echo "Failed to remove the existing mender debian source from '/etc/apt/sources.list'."
-            echo "This probably means that there already exists a source in your sources.list."
-            echo "Please remove it manually before proceeding."
-            exit 1
-        fi
+    repo_deprecated="deb [arch=$ARCH] $REPO_URL $CHANNEL main"
+    if grep -F "$repo_deprecated" /etc/apt/sources.list; then
```

Comment:
Maybe redirect stdout to /dev/null to avoid polluting the console.

```suggestion
    if grep -F "$repo_deprecated" /etc/apt/sources.list >/dev/null; then
```

Or, if you prefer, use `-q`:

```suggestion
    if grep -q -F "$repo_deprecated" /etc/apt/sources.list; then
```

Reply:
Thanks